UI/index.html

```diff
@@ -0,0 +1,524 @@
+                                <span class="fa fa-star checked"></span>
+                                <span class="fa fa-star checked"></span>
+                                <span class="fa fa-star"></span>
+                                <span class="fa fa-star"></span>
+                                <span class="fa fa-star"></span>
+                            </div>
+                            <p class="vendor-name">Vendor: GYM24</p>
+                            <p>Link for more info:<a class="vendor-link" href="#">https://www.gym24.by/</a></p>
+                            <p class="description-offer">Short description: GYM24 is the first autonomous 24-hour gym in the center of Minsk,
+                                which offers everyone training in a format convenient for them at a minimal cost.</p>
+                            <p class="hashtags-offer">Hashtags:
+                                <span class="tag">#gym</span>
+                                <span class="tag">#sport</span>
+                                <span class="tag">#health</span>
+                                <span class="tag">#minsk</span>
+                                <span class="tag">#lovesport</span>
+                                <span class="tag">#activelife</span>
+                                <span class="tag">#goodmood</span>
+                            </p>
+                            <div class="feedbacks offer-info">
+                                <h4>Feedbacks</h4>
+                                <p class="review">Name:
+                                    <span class="review">The best gym in Minsk!</span>
+                                </p>
+                                <button class="more-feedbacks-button">More feedbacks</button>
+                            </div>
+
+                        </div>
+                    </div>
+                    <div class="offer-footer">
+                        <button class="write-feedback-button"><img src="feedback.png" class="feedback-image"></button>
+                    </div>
+                </div>
+
+                <div class="offer">
+                    <div class="offer-header">
+                        <div class="offer-picture">
+                            <div class="image-percent-valid">
+                                <img src="picture.png" class="offer-image">
+                                <p class="percent">20%</p>
+                                <p class="valid-until">Until 01.04.2021</p>
+                            </div>
+                        </div>
+
+                        <div class="offer-info">
+                            <h3 class="offer-label">Unlimited monthly gym membership</h3>
+                            <div class= "rating">
+                                <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
```

Comment:
There's no need to import these styles in each article, better to put this import to the document`s head.